src/index.js

```diff
@@ -1,53 +1,50 @@
 const React = require("react");
 const ReactDOM = require("react-dom");
+const PropTypes = require("prop-types");
 
-function noop() {}
+class Clipboard extends React.Component {
 
-const Clipboard = React.createClass({
+  static propTypes = {
+    value : PropTypes.string.isRequired,
+    className : PropTypes.string,
+    style : PropTypes.object,
+    onCopy : PropTypes.func
+  };
 
-  propTypes: {
-    value : React.PropTypes.string.isRequired,
-    className : React.PropTypes.string,
-    style : React.PropTypes.object,
-    onCopy : React.PropTypes.func
-  },
-
-  getDefaultProps() {
-    return {
-      className : "clipboard",
-      style : {
-        position : "fixed",
-        overflow : "hidden",
-        clip     : "rect(0 0 0 0)",
-        height   : 1,
-        width    : 1,
-        margin   : -1,
-        padding  : 0,
-        border   : 0
-      },
-      onCopy : noop
-    };
-  },
+  static defaultProps = defaultProps = {
```

Comment:
This line should be:

```js
static defaultProps = {
```

Reply:
Oops that's a typo!

Fixed in 18ef11e724d8c24e71b7580841b5d8271e0d4dc2.